<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addElllipse</class>
 <widget class="QDialog" name="addElllipse">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>380</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="title">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Ellipse Creator&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="shapeDimensions">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>351</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>Shape Dimensions</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>328</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="point">
       <property name="text">
        <string>Point(x,y)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="x"/>
     </item>
     <item>
      <widget class="QLineEdit" name="y"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>174</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="length">
       <property name="text">
        <string>Semi Major Axis</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lengthBox"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>188</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="width">
       <property name="text">
        <string>Semi Minor Axis</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="widthBox"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="brushProperties">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>320</y>
     <width>301</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>Brush Properties</string>
   </property>
   <widget class="QWidget" name="layoutWidget_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>132</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="QLabel" name="brushColor">
       <property name="text">
        <string>Brush Color</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_3">
       <item>
        <property name="text">
         <string>---</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>White</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Black</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Red</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Green</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Blue</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Cyan</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Magenta</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Yellow</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Gray</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>146</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <widget class="QLabel" name="brushStyle">
       <property name="text">
        <string>Brush Style</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_4">
       <item>
        <property name="text">
         <string>---</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SolidPattern</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>HorPattern</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>VerPattern</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>NoBrush</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="penProperties">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>351</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>Pen Properties</string>
   </property>
   <widget class="QWidget" name="layoutWidget_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>123</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="penColor">
       <property name="text">
        <string>Pen Color</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="penColorBox">
       <item>
        <property name="text">
         <string>---</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>White</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Black</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Red</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Green</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Blue</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Cyan</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Magenta</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Yellow</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Gray</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>90</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="penWidth">
       <property name="text">
        <string>Pen Width</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="penWidthBox">
       <property name="maximum">
        <number>20</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>156</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="penStyle">
       <property name="text">
        <string>Pen Style</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="penStyleBox">
       <item>
        <property name="text">
         <string>---</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>NoPen</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SolidLine</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>DashLine</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>DotLine</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>DashDotLine</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>DashDotDotLine</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>154</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="penCapStyle">
       <property name="text">
        <string>Pen Cap Style</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox">
       <item>
        <property name="text">
         <string>---</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>FlatCap</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SquareCap</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>RoundCap</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QLabel" name="penJoinStyle">
       <property name="text">
        <string>Pen Join Style</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_2">
       <item>
        <property name="text">
         <string>---</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>MiterJoin</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>BevelJoin</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>RoundJoin</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>addElllipse</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>addElllipse</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
